unit_test.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+cd $APP_ROOT
+
+# Deploy ephemeral DB
+source $CICD_ROOT/deploy_ephemeral_db.sh
+
+# Map env vars set by `deploy_ephemeral_db.sh` if vars the app users are different
+export POSTGRES_ADMIN_PASSWORD=$DATABASE_ADMIN_PASSWORD
+export DB_HOST=$DATABASE_HOST
+export DB_DATABASE=$DATABASE_NAME
+export DB_USERNAME=$DATABASE_USER
+export DB_PASSWORD=$DATABASE_PASSWORD
+export DB_PORT=$DATABASE_PORT
```

Comment:
Just looked in the config for Remediations, looks like the env variables for db for tests are as follows:
```
DB_USERNAME = postgres
DB_PASSWORD = remediations
DB_DATABASE = remediations
DB_HOST = 127.0.0.1
```

Reply:
cool. I think we're good to go here. The ephemeral DB sets its own stuff so we'll just use those to match